### ISSUE TITLE
Import images

### ISSUE THREAD

**Issue by stefanbeller** [mapeditor/tiled#180]
This is the semi-reverse of 'save as image'. 
(Moving https://sourceforge.net/apps/mantisbt/tiled/view.php?id=17 here)

> I am involved in a huge community, where "mapping" ist part of the fanwork section
> ( http://fanart.pokefans.net/maps/  ). In addition to map editors, we use image editing 
> programs like Paint/Gimp/Fireworks (some work cannot be done with map editors). 
> In order to edit them again, images need to be imported. Of course, this could be 
> done by importing those maps as a tileset - nevertheless, a proper import-function 
> would be much more convenient (the image should be imported as a single layer).

Related to  #169:
I wonder if "Import Image" should be a toplevel functionality (such as layer menu->Import Image into Tilelayer)
or if it would be better to first import an image into an image layer and then
converting that image layer to a tile layer by adding the image as a tileset and filling the map with the tiles accordingly.

**Comment by bjorn** [comment]
Functionality requested on the forum multiple times:

* https://discourse.mapeditor.org/t/opening-anothers-map/2644
* https://discourse.mapeditor.org/t/help-with-lost-tiled-map-plz/3267

**Comment by bjorn** [comment]
Of course, we also have the [Import from Image](https://github.com/bjorn/tiled/wiki/Import-from-Image) wiki page, which links to a Python script for doing this.
